src/assets/scss/_resizable.scss

```diff
@@ -0,0 +1,17 @@
+.divider-hitbox {
+    cursor: col-resize;
+    align-self: stretch;
+    display: flex;
+    align-items: center;
+    padding: 0 1.5rem 0 0.4rem;
+    top: 50%;
+    color: #ccc;
```

Comment:
`color: #bbb;` is defined after this style and should be the one we use. Let's remove `color: #ccc`

Reply:
fixed